generate_options.json

```diff
@@ -8,8 +8,5 @@
     "--use-onever",
     "--preview-chk",
     "--version=V2"
-  ],
-  "afterScripts": [
-    "gofmt -w ./services/"
```

Comment:
The formatting happens elsewhere yes?

Reply:
yeah, I just embedded this into the script - before we generate the changelog. And I remove this since we do not actually have a place that is using this right now